api/api.go

```diff
@@ -198,8 +197,8 @@ func (c *Client) Submit(iter *Iteration) (*Submission, error) {
 }
 
 // List available problems for a language.
-func (c *Client) List(language string) ([]string, error) {
-	url := fmt.Sprintf("%s/tracks/%s", c.XAPIHost, language)
+func (c *Client) List(trackID string) ([]string, error) {
+	url := fmt.Sprintf("%s/tracks/%s", c.XAPIHost, trackID)
```

Comment:
I don't want to change variable names for things that are not relevant to the Status command. That mixes in unrelated changes, and it makes the git history more confusing (and the code harder to review).